pkg/controller/core.oam.dev/v1alpha2/setup.go

```diff
@@ -17,13 +17,14 @@ limitations under the License.
 package v1alpha2
 
 import (
+	"github.com/crossplane/crossplane-runtime/pkg/logging"
 	ctrl "sigs.k8s.io/controller-runtime"
 
-	"github.com/oam-dev/kubevela/pkg/controller/core.oam.dev/v1alpha2/application"
-
-	"github.com/crossplane/crossplane-runtime/pkg/logging"
+	"github.com/oam-dev/kubevela/pkg/controller/core.oam.dev/v1alpha2/core/traits/traitdefinition"
+	"github.com/oam-dev/kubevela/pkg/controller/core.oam.dev/v1alpha2/core/workloads/workloaddefinition"
 
```

Comment:
why leave an empty line here? And I have commented before, you close my comment with no reply.

Reply:
Fixed, thanks.